frontend/packages/api_client/src/models/OpenEdXAppServer.ts

```diff
@@ -84,6 +84,30 @@ export interface OpenEdXAppServer {
      * @memberof OpenEdXAppServer
      */
     edxPlatformCommit: string;
+    /**
+     * The repository to pull the default Ansible playbook from.
+     * @type {string}
+     * @memberof OpenEdXAppServer
+     */
+    ansibleAppserverRepoUrl?: string;
+    /**
+     * The path to the common appserver playbook to run on all appservers.
+     * @type {string}
+     * @memberof OpenEdXAppServer
+     */
+    ansibleAppserverPlaybook?: string;
+    /**
+     * The path to the requirements file for the common appserver playbook.
+     * @type {string}
+     * @memberof OpenEdXAppServer
+     */
+    ansibleAppserverRequirementsPath?: string;
+    /**
+     * The version of the Ansible playbook repository to checkout.
+     * @type {string}
+     * @memberof OpenEdXAppServer
+     */
+    ansibleAppserverVersion?: string;
```

Comment:
@giantas, just in case. I just merged a PR that adds the same fields in the client, so you need to regenerate it.

Reply:
@0x29a thanks for the heads-up. There doesn't seem to be a diff currently.